include/odp/api/spec/std_types.h

```diff
@@ -32,6 +32,13 @@ extern "C" {
  */
 
 /**
+ * @typedef odp_percent_t
+ * Use odp_percent_t for specifying fields that are percentages. It is a fixed
+ * point integer whose units are expressed as one-hundredth of a percent.
+ * Hence 100% is represented as integer value 10000.
+ */
+
```

Comment:
Since API specifies already what odp_percent_t is, it's better do also the typedef here. Also the documentation should say that it's _unsigned_ integer. May be uint32_t is safer choice than uint16_t, so that percent calculations do not easily wrap around.

Reply:
I can move the typedef here.
Not sure if I understand the need to move to uint32_t, I have only defined 100% as 10,000 and we should be fine with uin16_t. Any other specific reason for using uint32_t?

Reply:
Replace "@typedef odp_percent_t" doxygen tag with the short comment line ("Percent type"). The tag is not needed anymore as the typedef follows the comment.

Also, remove the extra line feed.